backend/src/test/java/gov/cdc/usds/simplereport/service/BaseServiceTest.java

```diff
@@ -107,7 +108,8 @@ protected void clearOktaUsers() {
     protected void clearOktaGroups() {
         for (Group g : _oktaClient.listGroups()) {
             String groupName = g.getProfile().getName();
-            if (groupName.startsWith(_authorizationProperties.getRolePrefix())) {
+            if (g.getType() == GroupType.OKTA_GROUP &&
+                        groupName.startsWith(_authorizationProperties.getRolePrefix())) {
```

Comment:
the same change should be made in BaseApiTest.java (though given the prognosis for Okta remaining in integration tests, maybe it doesn't matter too too much anyway)

Reply:
Any changes you think are needed when we roll forward and want to add to the branch, I'm fine with—I just restored the things that you had to back out to make the older API version work. 😄